src/pages/documentation/release-notes/index.md

```diff
@@ -4,456 +4,16 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
```

Comment:
I renamed the old "index.md" to "2022.md" then created a new "index.md" file. All the red "removed" lines are actually copied to the file "2022.md".